### PR TITLE
[Dungeon] Shattered Halls

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

**Comment by Geekena** [comment]
damage and spells are ok
heroic aswell
did not found any problem

File: data/sql/updates/pending_db_world/rev_1588338801814673000.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1081, `maxdmg` = 1531, `DamageModifier` = 1.01 WHERE `entry` = 16807;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1120, `maxdmg` = 1584, `DamageModifier` = 1.01 WHERE `entry` = 20568;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 918, `maxdmg` = 1302, `DamageModifier` = 1.01 WHERE `entry` = 16809;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1178, `maxdmg` = 1670, `DamageModifier` = 1.01 WHERE `entry` = 20596;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1129, `maxdmg` = 1601, `DamageModifier` = 1.01 WHERE `entry` = 16808;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1178, `maxdmg` = 1670, `DamageModifier` = 1.01 WHERE `entry` = 20597;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1254, `maxdmg` = 1778, `DamageModifier` = 1.01 WHERE `entry` = 20923;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1609, `maxdmg` = 2281, `DamageModifier` = 1.01 WHERE `entry` = 20993;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338801814673000.sql:61]: update typeflags pls

Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1588338801814673000.sql:61]: Correct the flags assignment like the wiki describe https://github.com/azerothcore/wiki/blob/master/docs/Dealing-with-SQL-files.md#flags

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1588338801814673000.sql]: Sorted.

**Pull request merged**